scripts/qemu/run-tests.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+# Copyright (C) 2024 Intel Corporation
+# Under the Apache License v2.0 with LLVM Exceptions. See LICENSE.TXT.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+set -x
+set -e
+
```

Comment:
same for:
```
# Set ptrace value for IPC test
echo password | sudo bash -c "echo 0 > /proc/sys/kernel/yama/ptrace_scope"
```

it resets to default value after shutdown

Reply:
done